### ISSUE TITLE
Font error using LuaLaTeX

### ISSUE THREAD

**Issue by Duval23** [matze/mtheme#187]
I installed both the metropolis theme and fira fonts using Tex Live and TeXstudio as editor. While compiling my template using XeLaTeX works without problems, I receive the following error using LuaLaTeX:
```
Font shape `EU2/FiraSansLight(0)/m/it' undefined(Font) using `EU2/FiraSansLight(0)/m/n' instead
Some font shapes were not available, defaults substituted.
```

This is my sort of 'minimal example' where the error occurs:

```
\documentclass[10pt]{beamer}
\usetheme{metropolis}
\begin{document}
\section{Gliederungspunkt}
\end{document}
```

At first I thought of wrong fonts, but as XeLaTeX works fine, I'm really confused. Any thoughts on that?

**Comment by benjamin-weiss** [comment]
The font loading mechanisms for XeTeX and LuaTeX are different. And LuaTeX doesn't seem to find the italic shape automatically. So it is definitely a bug in the theme. Until it is fixed you can add the following line after loading loading the theme:
```TeX
\setsansfont[BoldFont={Fira Sans}, ItalicFont={Fira Sans Light Italic}]{Fira Sans Light}
```

**Comment by rchurchley** [comment]
Very strange! As far as I can tell, this is a bug in LuaLaTeX itself. The problem seems to arise whenever `\setsansfont{...}` is called with a font which is itself a weight-variant of another font. At least, I can reproduce the problem very easily without using any part of ᴍᴇᴛʀᴏᴘᴏʟɪs:

```latex
\documentclass{article}
\usepackage{fontspec}
\setmainfont{Helvetica Neue UltraLight}[BoldFont={Helvetica Neue}]

\begin{document}

    Regular
    \textit{Italic}
    \textbf{Bold}
    \textbf{\textit{Bold Italic}}

\end{document}
```

I suspect our only option is to work around it as @benjamin-weiss suggests. I've submitted the fix as #188.
